devel/simple-web-server/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.1
+
+name	  	    simple-web-server
+version		    3.0.0-rc3
+categories          devel
+license             MIT
+maintainers         gmail.com:rene.vanpaassen @repagh openmaintainer
```

Comment:
You have to put brackets (`{` and `}`) around your email and GitHub handle to associate them.

Reply:
done & squashed.